test/src/test/kotlin/graphql/nadel/tests/EngineTests.kt

```diff
@@ -44,6 +44,7 @@ private val singleTestToRun = (System.getenv("TEST_NAME") ?: "")
  * NadelExecutionInput.Builder#transformExecutionHints method in your test hook class.
  */
 private val defaultHints = NadelExecutionHints.newHints()
+    .newDocumentCompiler { true }
```

Comment:
Tests have already been updated to new compiler, so I'm defaulting this to true. One thing to note, when we eventually remove the variables FF, we'll need to update all our tests.